packages/@react-spectrum/combobox/docs/ComboBox.mdx

```diff
@@ -0,0 +1,767 @@
+      <p>
+        Please indicate what fruit you would like included with your delivery. If your desired choice does not appear in the list
+        feel free to write your own selection.
+      </p>
+      <ComboBox
+        label="Preferred fruit"
+        defaultItems={options}
+        allowsCustomValue>
+        {item => <Item key={item.name}>{item.name}</Item>}
+      </ComboBox>
+    </>
+  );
+}
+```
+
+## Labeling
+ComboBox can be labeled using the `label` prop. If the ComboBox is a required field, the `isRequired` and `necessityIndicator` props can be used to show a required state.
+
+```tsx example
+<ComboBox label="Favorite Animal" isRequired necessityIndicator="icon">
+  <Item key="red panda">Red Panda</Item>
+  <Item key="cat">Cat</Item>
+  <Item key="dog">Dog</Item>
+  <Item key="aardvark">Aardvark</Item>
+  <Item key="kangaroo">Kangaroo</Item>
+  <Item key="snake">Snake</Item>
+</ComboBox>
+```
+```tsx example
+<ComboBox label="Favorite Animal" isRequired necessityIndicator="label">
+  <Item key="red panda">Red Panda</Item>
+  <Item key="cat">Cat</Item>
+  <Item key="dog">Dog</Item>
+  <Item key="aardvark">Aardvark</Item>
+  <Item key="kangaroo">Kangaroo</Item>
+  <Item key="snake">Snake</Item>
+</ComboBox>
+```
+```tsx example
+<ComboBox label="Favorite Animal" necessityIndicator="label">
+  <Item key="red panda">Red Panda</Item>
+  <Item key="cat">Cat</Item>
+  <Item key="dog">Dog</Item>
+  <Item key="aardvark">Aardvark</Item>
+  <Item key="kangaroo">Kangaroo</Item>
+  <Item key="snake">Snake</Item>
+</ComboBox>
+```
+
+### Accessibility
+If a visible label isn't specified, an `aria-label` must be provided to the ComboBox for
+accessibility. If the field is labeled by a separate element, an `aria-labelledby` prop must be provided using
+the `id` of the labeling element instead.
+
+### Internationalization
+In order to internationalize a ComboBox, a localized string should be passed to the `label` or `aria-label` prop.
+When the `necessityIndicator` prop is set to `"label"`, a localized string will be provided for `"(required)"` or `"(optional)"` automatically.
+
+## Selection
+Setting a selected option can be done by using the `defaultSelectedKey` or `selectedKey` prop. The selected key corresponds to the `key` of an item. See [Events](#events) for more details on selection events.
+
+```tsx example
+function Example() {
+  let options = [
+    {id: 1, name: 'Adobe Photoshop'},
+    {id: 2, name: 'Adobe XD'},
+    {id: 3, name: 'Adobe InDesign'},
+    {id: 4, name: 'Adobe AfterEffects'},
+    {id: 5, name: 'Adobe Illustrator'},
+    {id: 6, name: 'Adobe Lightroom'},
+    {id: 7, name: 'Adobe Premiere Pro'},
+    {id: 8, name: 'Adobe Fresco'},
+    {id: 9, name: 'Adobe Dreamweaver'}
+  ];
+  let [productId, setProductId] = React.useState('9');
+
+  return (
+    <Flex gap="size-150" wrap>
+      <ComboBox
+        label="Pick an Adobe product (uncontrolled)"
+        defaultItems={options}
+        defaultSelectedKey="9">
```

Comment:
I think this isn't working. Maybe needs to be a number instead of a string?

```suggestion
        defaultSelectedKey={9}>
```

Reply:
Derp good catch on the broken example, fixing.